src/dreyfus/src/dreyfus_rpc.erl

```diff
@@ -102,10 +102,12 @@ disk_size(DbName, DDoc, IndexName) ->
     end.
 
 get_or_create_db(DbName, Options) ->
+    DbOpts = mem3_shards:opts_for_db(DbName),
+    Options1 = mem3_util:merge_opts(DbOpts, Options),
     case couch_db:open_int(DbName, Options) of
```

Comment:
Should we use `couch_db:open_int(DbName, Options1)` here?

Reply:
Yeap definitely hahaha, good catch.